client/src/helpers/sortObjectKeys.js

```diff
@@ -0,0 +1,20 @@
+// transforms an object into an array
+
+export const sortedObjectKeys = (obj, sort) => {
+  return Object.keys(obj)
+    .sort(sort)
+    .map((key) => ({
+      key,
+      value: obj[key]
+    }))
+}
+
+export const sortedObjectKeysByValues = (obj, ascending = true) => {
```

Comment:
this might be able to be reworked to handle sorting numerical and alphabetical values like `1-9a-z`

Reply:
added a note that it expects all numerical values in the object... maybe we can improve this or add another method to check if the keys are homogeneous or how to handle them